### ISSUE TITLE
[featreq] support for youtube

### ISSUE THREAD

**Issue by lavenderdotpet** [laylavish/uBlockOrigin-HUGE-AI-Blocklist#95]
### i imagine it would likely be to much work so nw if not

**Comment by laylavish** [comment]
No, sorry at this time I won't... just kidding! This is something that I've been doing for the past five months via [blocktube](https://github.com/amitbl/blocktube). It's not done yet, but when it is, it will have it's own repo just like this one. I'm hoping to get it done by the time the new year rolls out, if not shortly after that.

**Comment by Binocularbath** [comment]
> No, sorry at this time I won't... just kidding! This is something that I've been doing for the past five months via [blocktube](https://github.com/amitbl/blocktube). It's not done yet, but when it is, it will have it's own repo just like this one. I'm hoping to get it done by the time the new year rolls out, if not shortly after that.

Hello!

Did you end up releasing the yt ai block list? I'm very interested.

**Comment by laylavish** [comment]
> > No, sorry at this time I won't... just kidding! This is something that I've been doing for the past five months via [blocktube](https://github.com/amitbl/blocktube). It's not done yet, but when it is, it will have it's own repo just like this one. I'm hoping to get it done by the time the new year rolls out, if not shortly after that.
> 
> Hello!
> 
> Did you end up releasing the yt ai block list? I'm very interested.

I've been pretty busy lately, but I think I'm nearing the end of the list. I'm hoping by the time it's finished that blocktube implements a subscription feature for lists, because right now you have to update everything manually. That's one of the big reasons on why this has been taking so long; I want this to come out swinging with a huge list because that initial release will probably be the only one that people will see/use. I'm willing to bet it's unlikely that people will periodically update the list manually every time there is an update. Plus, I need to finish the repo for the list anyways. So it'll be a bit more time until it's out, but it's coming.
